### PR TITLE
:memo: Docs for interoperability with Firebase

### PR DESCRIPTION
**Pull request by jonas-m-**
## Description
Adding a drop of knowledge when working with Firebase plugins

## Related Issue
https://github.com/phonegap/phonegap-plugin-push/issues/929

## Motivation and Context
As suggested by @fredgalvao

## How Has This Been Tested?
Not applicable

## Screenshots (if appropriate):
Not applicable

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [-] My code follows the code style of this project.
- [-] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [-] I have added tests to cover my changes.
- [-] All new and existing tests passed.

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -114,6 +115,30 @@ or
...
+
+Problems may arise when push plugin is used along plugins, that implement Firebase functionality (cordova-plugin-firebase-analytics, for example). Firebase uses `@string/google_app_id`, as does the push plugin, though the value format differs, causing problems like this: `Invalid google_app_id. Firebase Analytics disabled`.
+
+To make the two work together, you need to migrate your GCM project from Google console to Firebase console:
+
+1) In Firebase console - [import your existing GCM project](https://firebase.google.com/support/guides/google-android#migrate_your_console_project), don't create a new one.
+2) Set your `SENDER_ID` variable to match the id of your imported Firebase project. In case of cordova, your `config.xml` would look something like this:
+```xml
+<plugin name="phonegap-plugin-push" spec="~1.10.0">
+    <variable name="SENDER_ID" value="1:956432534015:android:df201d13e7261425" />
```
Comment (fredgalvao) [inline docs/INSTALLATION.md]: Are those values sensitive information in any way? If so, I'd recommend using dummy values, just to demonstrate the overall format of the values without actually providing a valid one. Specially if those are yours :wink:

**Comment by macdonst** [comment]
@jonas-m- did you do any testing with GCM with a SENDER_ID in FCM format?

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -114,6 +115,30 @@ or
 cordova plugin add --save cordova-plugin-facebook4 --variable APP_ID="App ID" --variable APP_NAME="App Name"
 ```
 
+### Co-existing with plugins that use Firebase
+
+Problems may arise when push plugin is used along plugins, that implement Firebase functionality (cordova-plugin-firebase-analytics, for example). Firebase uses `@string/google_app_id`, as does the push plugin, though the value format differs, causing problems like this: `Invalid google_app_id. Firebase Analytics disabled`.
```
Comment (fredgalvao) [inline docs/INSTALLATION.md]: `along plugins, that implement` -> `along plugins that implement` =  personal suggestion to increase readability.

File: docs/INSTALLATION.md (hunk 1 above)
Comment (jonas-m-) [inline docs/INSTALLATION.md]: These are not sensitive, nor they are mine (or anyone else's, for that matter) - just dummy values while preserving the real format :)

**Comment by jonas-m-** [comment]
@macdonst Yes, I did no changes on the back-end side (using GCM gateway) and the messages are still getting delivered. I could test it one more time, cleaning-up all of the old tokens, so that we could be sure, though, IMO, that is unlikely to cause any problems

File: docs/INSTALLATION.md (hunk 2 above)
Comment (jonas-m-) [inline docs/INSTALLATION.md]: Changed

**Comment by jonas-m-** [comment]
@macdonst Tested again - does really work :)

**Comment by macdonst** [comment]
@jonas-m- thanks for the PR. I have to rebase the 2.0.0 branch and get the FCM code out to everyone in the next couple of weeks.

**Pull request merged**